pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/generic/MultiVersionSchemaInfoProvider.java

```
@@ -71,14 +70,32 @@ public GenericSchema getSchema(byte[] schemaVersion) {
         }
     }
 
-    private GenericSchema loadSchema(byte[] schemaVersion) throws ExecutionException, InterruptedException {
-        Optional<SchemaInfo> schemaInfo = pulsarClient.getLookup()
-                .getSchema(topicName, schemaVersion).get();
-        return schemaInfo.map(GenericSchemaImpl::of).orElse(null);
+    @Override
+    public SchemaInfo getLatestSchema() {
+        try {
+            Optional<SchemaInfo> optional = pulsarClient.getLookup()
+                    .getSchema(topicName).get();
+            return optional.orElse(null);
+        } catch (ExecutionException e) {
+            LOG.error("Can't get current schema for topic {}",
+                    topicName.toString(), e);
+            return null;
```

I think we should throw some runtime exception instead of returning `null`.